docs/_config.yml

```diff
@@ -22,3 +22,12 @@ aux_links:
 
   "OpenAPI Specification":
     - "/spec"
+
+compress_html:
```

Comment:
What is this for?  I assume I should carry forward these changes too?

Reply:
This was necessary to avoid invalid HTML when enabling line numbers for the code fences https://just-the-docs.com/docs/ui-components/code/line-numbers/